schema/string_test.go

```diff
@@ -16,44 +16,44 @@ func TestDecodeString_InvalidUUIDVersion(t *testing.T) {
 	}
 }
 
-func TestDecodeString_Success(t *testing.T) {
-	t.Run("ValidMinLength", func(t *testing.T) {
-		var data = []struct {
-			Desc        string
-			Value       string
-			Format      string
-			constraints Constraints
-		}{
-			{"URI", "http://google.com", stringURI, Constraints{MinLength: 5}},
-			{"Email", "foo@bar.com", stringEmail, Constraints{MinLength: 3}},
-			{"UUID", "C56A4180-65AA-42EC-A945-5FD21DEC0538", stringUUID, Constraints{MinLength: 36}},
-		}
-		for _, d := range data {
-			v, err := decodeString(d.Format, d.Value, d.constraints)
-			if err != nil {
-				t.Errorf("want:nil got:%q", err)
-			}
-			if v != d.Value {
-				t.Errorf("want:%s got:%s", d.Value, v)
+func TestDecodeString_ErrorCheckingConstraints(t *testing.T) {
+	data := []struct {
+		desc        string
+		value       string
+		format      string
+		constraints Constraints
+	}{
+		{"InvalidMinLength", "6fa459ea-ee8a-3ca4-894e-db77e160355e", stringUUID, Constraints{MinLength: 100}},
```

Comment:
These descriptors should not have the same name. That would make harder to spot the case when the subtest fails. Would you mean InvalidMinLength_UUID and analogous changes for the other ones?

Reply:
Ok, I will change descriptions to different name. i.e
InvalidMinLength_UUID
InvalidMinLength_URL
InvalidMinLength_Email

Reply:
Thanks!